apps/recovery-portal/src/main/webapp/username-recovery-tenant-request.jsp

```diff
@@ -71,9 +72,19 @@
 
             <div class="segment-form">
                 <form class="ui large form" method="post" action="recoverusername.do" id="tenantBasedRecovery">
+                    <%
+                        if (!IdentityTenantUtil.isTenantQualifiedUrlsEnabled()) {
+                    %>
                     <input id="tenant-domain" type="text" name="tenantDomain"
                                 class="form-control ">
-                        
+                    <%
+                        } else {
+                    %>
+                    <input id="tenant-domain" type="hidden" name="tenantDomain" value='<%=Encode.forHtmlAttribute
+                                (IdentityTenantUtil.getTenantDomainFromContext())%>' class="form-control">
+                    <%
```

Comment:
Why tenant domain had to be passed as a parameter in this case

Reply:
fixed